### PR TITLE
Add reporting system to admin panel

### PR DESCRIPTION
**Pull request by ymasiuk**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ default: &default
 
 development:
   <<: *default
-  database: peatio_development
```
Comment (ysv) [inline config/database.yml]: What ?

**Review by ysv — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 1 above)
Comment (ymasiuk) [reply config/database.yml]: @ysv fixed

**Comment by ysv** [comment]
screen shot 2018-09-18 at 4 42 15 pm

File: app/views/admin/accountments/_trade.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<tr class="table td primary">
+  <td><%= trade.id %></td>
+  <td><%= link_to Member.find_by_id(trade.bid_member_id).email, admin_member_path(trade.bid_member_id), target: '_blank' %></td>
+  <td><%= link_to Member.find_by_id(trade.ask_member_id).email, admin_member_path(trade.ask_member_id), target: '_blank' %></td>
```
Comment (tamadamas) [inline app/views/admin/accountments/_trade.html.erb:4]: it's very bad to do any db queries in the view level!

**Review by tamadamas — CHANGES_REQUESTED** [review]

File: app/controllers/admin/accountments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    def withdraws
+      @withdraws = Withdraw.all.order('id desc').page(params[:page]).per(20)
+    end
+
+    def members
+      @members = Member.all.order('id desc').page(params[:page]).per(20)
+    end
+
+    def trades
+      @trades = Trade.all.order('id desc').page(params[:page]).per(20)
```
Comment (tamadamas) [inline app/controllers/admin/accountments_controller.rb]: `20` should be constant

**Review by tamadamas — CHANGES_REQUESTED** [review]

**Pull request closed without merging**